app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -218,7 +218,9 @@ const SendBitcoinDestinationScreen = ({
   const contacts = useMemo(() => data?.me?.contacts ?? [], [data?.me?.contacts])
 
   const { LL } = useI18nContext()
-  const [userDefaultWalletIdQuery] = useUserDefaultWalletIdLazyQuery()
+  const [userDefaultWalletIdQuery] = useUserDefaultWalletIdLazyQuery({
+    fetchPolicy: "network-only",
```

Comment:
do we still want to cache it? maybe we could go for `no-cache`?

Reply:
Switching to that